File: MusikPlayer8/app/src/main/java/com/freesongs/musikplayer8/model/BaiHat.java
package com.freesongs.musikplayer8.model;

public class BaiHat {

    private String tenbaihat, tacgia, baihat, hinhanh, thoiluong;

    public BaiHat(String tenbaihat, String tacgia, String baihat, String hinhanh, String thoiluong) {
        this.tenbaihat = tenbaihat;
        this.tacgia = tacgia;
        this.baihat = baihat;
        this.hinhanh = hinhanh;
        this.thoiluong = thoiluong;
    }

    public String getTenbaihat() {
        return tenbaihat;
    }

    public void setTenbaihat(String tenbaihat) {
        this.tenbaihat = tenbaihat;
    }

    public String getTacgia() {
        return tacgia;
    }

    public void setTacgia(String tacgia) {
        this.tacgia = tacgia;
    }

    public String getBaihat() {
        return baihat;
    }

    public void setBaihat(String baihat) {
        this.baihat = baihat;
    }

    public String getHinhanh() {
        return hinhanh;
    }

    public void setHinhanh(String hinhanh) {
        this.hinhanh = hinhanh;
    }

    public String getThoiluong() {
        return thoiluong;
    }

    public void setThoiluong(String thoiluong) {
        this.thoiluong = thoiluong;
    }

}

File: models/baihat.js
const mongoose = require("mongoose");
const Joi = require("joi");

const sodoBaiHat = new mongoose.Schema({
	tenbaihat: { type: String, required: true },
	tacgia: { type: String, required: true },
	baihat: { type: String, required: true },
	hinhanh: { type: String, required: true },
	thoiluong: { type: String, required: true },
});

const validate = (baihat) => {
	const schema = Joi.object({
		tenbaihat: Joi.string().required(),
		tacgia: Joi.string().required(),
		baihat: Joi.string().required(),
		hinhanh: Joi.string().required(),
		thoiluong: Joi.number().required(),
	});
	return schema.validate(baihat);
};

const BaiHat = mongoose.model("baihat", sodoBaiHat);

module.exports = { BaiHat, validate };